src/magellan_firmware/firmware/magellan_controller/Robot.cpp

```diff
@@ -43,7 +43,7 @@ void Robot::AutonomousInit() {
 
 void Robot::AutonomousPeriodic() {
     drivetrain_.SetThrottlePercent(throttle_percent_);
-    drivetrain_.SetSteeringAngle(-steering_angle_);
+    drivetrain_.SetSteeringAngle(steering_angle_);
```

Comment:
Does this break running on the car?

Reply:
This is the change you made to make the path follower work. This change matches Robot.cpp in the motion branch which is confirmed to work.